### ISSUE TITLE
Demodulation Floor Validation

### ISSUE THREAD

**Issue by johanstokking** [TheThingsNetwork/lorawan-stack#21]
**Summary:** 

The values in the demodulation floor table (in `pkg/networkserver/adr.go`) need to be verified. 

**What do you see now?**

The current table is based on what we have in v2, which in turn is based on the ADR issue https://github.com/TheThingsNetwork/ttn/issues/265#issuecomment-255092765, and some other sources.

**What do you want to see instead?**

We're pretty confident that the current values are correct (especially the ones already there in v2) but to be sure, we should validate the ones for SF6 data rates (if those will ever be used), and the ones for BW250/BW500.

**What can you do yourself and what do you need help with?**

We based the table on the information that we have. It would be helpful if people had an authoritive source of these values.

----

Original issue: https://github.com/TheThingsIndustries/lorawan-stack/issues/876 by @rvolosatovs

**Comment by johanstokking** [comment]
Needs input from a LoRa expert.

**Comment by johanstokking** [comment]
I just looked into this. I found a document from Semtech ([AN1200.22](https://www.frugalprototype.com/wp-content/uploads/2016/08/an1200.22.pdf)) which states that the noise floor scales linearly with 10*log10(B) in dBm where B is the bandwidth in Hz. A step from 125000 to 250000 or 250000 to 500000 is indeed a +3 dBm increase. So the current values are correct. I'll remove the TODO and close this issue.
